patgen/configvocab.go

```diff
@@ -131,12 +130,18 @@ func AddVocabDrift(mp Vocab, name string, rows int, pctDrift float32, copyFrom s
 	trow.CopyFrom(cprow)
 	nOn := NOnInTensor(cprow)
 	nDrift := int(math.Round(float64(nOn) * float64(pctDrift)))
-	nDrift = ints.MaxInt(1, nDrift) // ensure at least one
+	//nDrift = ints.MaxInt(1, nDrift) // ensure at least one //JWA, was uncommented
+	nDrift = ints.MaxInt(0, nDrift) // ensure at least one //JWA, new
```

Comment:
Here's a version that deals with "aliasing" for fractional drift levels, including case where drift < 1:

```Go
rmdr := 0.0 // remainder carryover in drift
drift := float64(nOn) * float64(pctDrift+derr) // precise fractional amount of drift
for i := 1; i < rows; i++ {
...
curDrift := math.Round(float64(nOn) * (float64(pctDrift)+rmdr))) // integer amount
nDrift = int(curDrift)
if nDrift > 0 {
  FlipBigts(trow, nDrift, nDrift, 1, 0)
}
rmdr += drift - curDrift // accumulate remainder
}
```
same logic as: https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm

Reply:
Is that a typo, 'FlipBigts'?